Orange/widgets/data/owimpute.py

```diff
@@ -346,6 +354,7 @@ def commit(self):
 
         if not self.data or not self.varmodel.rowCount():
             self.Outputs.data.send(self.data)
+            self.info.set_output_summary(self.info.NoOutput)
```

Comment:
When `self.data is not None` and `self.varmodel.rowCount() == 0`, there actually is data on the output.